### ISSUE TITLE
Error with using getAllCountries method in next.js 11

### ISSUE THREAD

**Issue by bfeldner** [manuelmhtr/countries-and-timezones#32]
We are using the getAllCountries method in our next.js v. 11 project, which stopped working when updating countries-and-timezones from 2.4 to 2.5.0.

Made a minimal example showing the error: https://codesandbox.io/s/nextjs-forked-5uj7q

**Comment by manuelmhtr** [comment]
Hi @bfeldner, thanks for reporting this problem.
Apparently the ESM build is only exporting a `default` (the whole package) but not individual functions.

As a quick fix, this should work:
```js
import ct from "countries-and-timezones";

const App = () => {
  const allCountries = ct.getAllCountries();
```

However, let me try other solutions to make `import { getAllCountries } from "countries-and-timezones";` works as well.

**Comment by bfeldner** [comment]
Hi @manuelmhtr , thank you for your response.

**Comment by manuelmhtr** [comment]
Hi @bfeldner,
I just released version `v3.0.1` which supports named ESM exports. Your original code should work with this version.
